src/Abc.Zebus.MessageDsl.Tests/MessageDsl/ParsedContractsTests.cs

```diff
@@ -498,6 +498,26 @@ public void should_return_source_text()
             attr.GetSourceTextInterval().ShouldEqual(new TextInterval(3, 11));
         }
 
+        [Test]
+        public void should_return_two_messages()
+        {
+            // Arrange
+            var name = "Command";
+            var genericParam = "IEnumerable";
+            var definitionText = "using System.Collections;" + Environment.NewLine +
```

Comment:
No need to have a fully valid definition here. A very simple one like `Foo(); Foo<T>();` would be better.
Also, `T` is better than `IEnumerable` :)